### PR TITLE
feat: Rare crops knowledge panel - Divinfood project

### PR DESCRIPTION
**Pull request by stephanegigandet**
New knowledge panel for "rare crops" of interest for the Divinfood project. This is just a start to show what can be done, we can improve on it later.

image

### CODE REVIEW

File: po/common/en.po [hunk 1]
```diff
@@ -6946,4 +6946,46 @@ msgstr "You cannot remove the main contact. Change it first."
...
+
+msgctxt "knowledge_panels_ingredients_rare_crops_what_you_can_do"
+msgid "<p>Consuming diverse cereals, legumes, vegetables and fruits contributes to:</p>"
+"<ul>"
+"<li>preserving neglected and underutilized species</li>"
+"<li>rural development and support of local farmers and companies</li>"
+"<li>preserving of diversified landscapes</li>"
+"<li>food security</li>"
+"<li>and your health!</li>"
+"</ul>"
```
Comment (teolemon) [inline po/common/en.po:6979]: could you break that down in strings that do not require HTML markup ?
Unrelated, We could use a list element in the KP syntax.

File: templates/api/knowledge-panels/health/ingredients/ingredients_rare_crops.tt.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "elements": [
+        {
+            "element_type": "text",
+            "text_element": {
+                "html":
+                `
+<b>[% lang("knowledge_panels_did_you_know") %]</b>
+<p>[% lang("knowledge_panels_ingredients_rare_crops_did_you_know") %]</p>
+<b>[% lang("knowledge_panels_what_you_can_do") %]</b>
```
Comment (teolemon) [inline templates/api/knowledge-panels/health/ingredients/ingredients_rare_crops.tt.json:21]: I would add a section about risks of low crop biodiversity.

Crop diversity: an insurance policy for food security. Just like a diverse investment portfolio protects your savings, a diverse range of crops protects our food supply from pests, diseases, and climate change. Monocultures are vulnerable, diversity is resilience.

File: po/common/en.po (hunk 1 above)
Comment (stephanegigandet) [reply po/common/en.po:6979]: We can, but there may not be enough context to have gramatically correct translations in all languages.

File: templates/api/knowledge-panels/health/ingredients/ingredients_rare_crops.tt.json (hunk 2 above)
Comment (stephanegigandet) [reply templates/api/knowledge-panels/health/ingredients/ingredients_rare_crops.tt.json:21]: I'll add a section about why it matters. I'm not too sure about the savings metaphor.

Comment (teolemon) [reply templates/api/knowledge-panels/health/ingredients/ingredients_rare_crops.tt.json:23]: me neither, a LLM artefact :-)

**Comment by stephanegigandet** [comment]
@teolemon Here is a new version:

image

The DVINFOOD project is supposed to come up with the right messages to inform consumers and encourage them to eat more rare crops, other partners are conducing surveys to understand what matters more to consumers. So this is mostly a placeholder at this point. It's best to keep it relatively short so that it fits well as a panel on the web and mobile.

**Review by teolemon — APPROVED** [review]
Good for a 1st version

**Pull request merged**